tests/integration/test_materialize_mysql_database/materialize_with_ddl.py

```diff
@@ -500,7 +500,8 @@ def err_sync_user_privs_with_materialize_mysql_database(clickhouse_node, mysql_n
     clickhouse_node.query(
         "CREATE DATABASE priv_err_db ENGINE = MaterializeMySQL('{}:3306', 'priv_err_db', 'test', '123')".format(
             service_name))
-
+    print("\n=== print log for CI test err ===\n")
+    print(clickhouse_node.exec_in_container(["bash", "-c", 'cat /var/log/clickhouse-server/clickhouse-server.log']))
```

Comment:
FYI you can find this log in `test_dir.tar` in integration tests report

Reply:
Oh this is too embarrassing.....